deps.edn

```diff
@@ -234,6 +234,8 @@
     methodical/methodical        {:mvn/version "0.15.1"}                    ; drop-in replacements for Clojure multimethods and adds several advanced features
     org.clj-commons/hickory      {:mvn/version "0.7.3"}                     ; Parse HTML into Clojure data structures
     org.clojure/algo.generic     {:mvn/version "0.1.3"}
+    peridot/peridot              {:git/url "https://github.com/piranha/peridot.git"
+                                  :sha "999d0a02425c906c35bace749654dc095ecf3e6a"} ; mocking Ring requests; update to upstream when/if it merges changes
```

Comment:
I'm thinking if I should just use Java lib directly? That would add about 20 lines of code in total, but one dependency less.

Reply:
OTOH, it's tested there and that's nice, so no, let it be dependency and I'll try to get my changes upstream.

Reply:
maybe you should link to a PR that you fixed the bug you need, or comment on why we're using your fork.